packages/react-jss/src/utils/sheets.js

```diff
@@ -0,0 +1,124 @@
+// @flow
+import warning from 'tiny-warning'
+import {getDynamicStyles, type StyleSheetFactoryOptions} from 'jss'
+import type {StyleSheet} from 'jss'
+import type {Context, DynamicRules, Styles} from '../types'
+import {getManager} from './managers'
+import {jss as defaultJss} from '../jss'
+import {addMetaForSheet, getMetaForSheet} from './sheets-meta'
+
+interface Options<Theme> {
+  context: Context;
+  theme: Theme;
+  name: string;
+  index: number;
+  styles: Styles<Theme>;
+  sheetOptions: $Diff<StyleSheetFactoryOptions, {index: number | void}>;
+}
+
+const getStyles = <Theme>(options: Options<Theme>) => {
+  const {styles} = options
+  if (typeof styles !== 'function') {
+    return styles
+  }
+
+  warning(
+    styles.length !== 0,
+    `[JSS] <${
+      options.name
+    } />'s styles function doesn't rely on the "theme" argument. We recommend declaring styles as an object instead.`
+  )
+
+  return styles(options.theme)
+}
+
+function getSheetOptions<Theme>(options: Options<Theme>, link: boolean) {
+  const classNamePrefix =
+    process.env.NODE_ENV === 'production' ? '' : `${options.name.replace(/\s/g, '-')}-`
```

Comment:
Since we started using a non-minified version in the other PR by default, it would make sense to do the same here, no?

Reply:
Yeah, but the change shouldn't be in this PR.